docs/uml/sequenceDiagrams/search/searchAll.puml

```diff
@@ -3,22 +3,21 @@
 !include ../../Style.puml
 
 autonumber
-
+hide footbox
 title Sequence Diagram of `search /all` Command
 
-actor   User
+
 participant "werkIt:WerkIt" AS WerkIt
 participant "ui:UI" AS UI1
 participant "parser:Parser" AS Parser
 participant "newCommand\n:SearchCommand" AS newCommand
 participant "planList\n:PlanList" AS PlanList
-participant "exerciseList\n:PlanList" AS ExerciseList
-participant "WorkoutList\n:PlanList" AS WorkoutList
+participant "exerciseList\n:ExerciseList" AS ExerciseList
+participant "workoutList\n:WorkoutList" AS WorkoutList
 
 activate WerkIt
 WerkIt -> UI1 : getUserInput()
 activate UI1
-User -> UI1 : `search /all` command
 WerkIt <-- UI1 : userInputTrimmed:String
 deactivate UI1
 
```

Comment:
For the uml diagrams, might want to consider simplifying your diagram by removing steps 1 to 4 (e.g. from getuserinput till the return Command.) Possibly standardize to the SD like the screenshot below as the reference frame will be used to reference a sequence diagram (not yet created) on a section called _Parsing User Input and Getting the Right Command_.

![image](https://user-images.githubusercontent.com/69451253/161319968-00628ff3-c259-42eb-91d1-bc7ddaadc46f.png)

Reply:
ehhhh, this belongs to @alanlowzies